config/dev1/sys.config

```diff
@@ -34,12 +34,11 @@
 
   {jobs,
    [{queues,
-     [{sync_jobs, [passive]},
-      {sync_workers, [{regulators,
-                       [{counter, [{limit, 10}]}
-                       ]},
-                      {producer, {aec_sync, sync_worker, []}}
-                     ]},
+     [{ping_workers, [{regulators, [{counter, [{limit, 3}]}]}]},
+      {mempool_workers, [{regulators, [{counter, [{limit, 1}]}]}]},
+      {sync_task_workers, [{regulators, [{counter, [{limit, 10}]}]}]},
+      {gossip_workers, [{regulators, [{counter, [{limit, 10}]}]}]},
```

Comment:
I would like to see markers that tell me these workers are sync workers. Perhaps prefix sync_ping_workers or something like it. Especially for mempool_workers which might be confusing.